<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>30</red>
        <green>30</green>
        <blue>30</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>61</red>
        <green>113</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>30</red>
        <green>30</green>
        <blue>30</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>61</red>
        <green>113</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>61</red>
        <green>113</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>61</red>
        <green>113</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-60</y>
      <width>1600</width>
      <height>1600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mainMenu">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>381</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="loadPuzzleBtn">
         <property name="text">
          <string>Load Puzzle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createManuallyBtn">
         <property name="text">
          <string>Create Board Manually</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="randomBoardBtn">
         <property name="text">
          <string>Create Random Board</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="documentationBtn">
         <property name="text">
          <string>Documentation</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="campingImage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>660</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/new/src/camping.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="randomBoard">
     <widget class="QPushButton" name="goBackBtn_4">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>50</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/new/src/back.png</normaloff>:/new/src/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QTableWidget" name="board3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>290</y>
        <width>661</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="216">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="216">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="216">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="solveBtn2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>190</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Solve!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateRandomBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>190</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Board</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearBoardBtn3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>190</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Board</string>
      </property>
     </widget>
     <widget class="QLabel" name="nightImage3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>690</width>
        <height>471</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::VLine</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/new/src/night.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loadPuzzle">
     <widget class="QPushButton" name="goBackBtn_3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>50</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/new/src/back.png</normaloff>:/new/src/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QTableWidget" name="board2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>290</y>
        <width>661</width>
        <height>471</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="216">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="216">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="216">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
     </widget>
     <widget class="QLabel" name="nightImage2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>690</width>
        <height>471</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::VLine</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/new/src/night.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="openFileBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="solveBtn3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>180</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Solve!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearBoardBtn2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Board</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="createManually">
     <widget class="QPushButton" name="goBackBtn_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>50</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/new/src/back.png</normaloff>:/new/src/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QTableWidget" name="board">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>289</y>
        <width>660</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="216">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="216">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="216">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
     </widget>
     <widget class="QLabel" name="nightImage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>290</y>
        <width>691</width>
        <height>471</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::VLine</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/new/src/night.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="clearBoardBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>140</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Clear Board</string>
      </property>
     </widget>
     <widget class="QLabel" name="rowsText">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>140</y>
        <width>124</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Rows&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="columnsText">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>170</y>
        <width>124</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Columns&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="generateBtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>140</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Board</string>
      </property>
     </widget>
     <widget class="QPushButton" name="solveBtn">
      <property name="geometry">
       <rect>
        <x>345</x>
        <y>200</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Solve!</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="rowsSpinBox">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="columnsSpinBox">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
     </widget>
     <zorder>nightImage</zorder>
     <zorder>goBackBtn_2</zorder>
     <zorder>board</zorder>
     <zorder>clearBoardBtn</zorder>
     <zorder>rowsText</zorder>
     <zorder>columnsText</zorder>
     <zorder>generateBtn</zorder>
     <zorder>solveBtn</zorder>
     <zorder>rowsSpinBox</zorder>
     <zorder>columnsSpinBox</zorder>
    </widget>
    <widget class="QWidget" name="documentation">
     <widget class="QLabel" name="documentationTitle">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DOCUMENTATION</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goBackBtn">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>50</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/new/src/back.png</normaloff>:/new/src/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
